test/test_quantized_tensor.py

```diff
@@ -298,28 +302,31 @@ def test_qtensor_view(self):
 
 
     def test_qtensor_reshape(self):
-        scale, zero_point, dtype = 1.0, 2, torch.quint8
-        q = torch._empty_affine_quantized([3, 5], scale=scale, zero_point=zero_point, dtype=dtype)
+        scale, zero_point, dtype = 1.0, 2, torch.uint8
+        q_int = torch.randint(0, 100, [3, 5], dtype=dtype)
+        q = torch._per_tensor_affine_qtensor(q_int, scale=scale, zero_point=zero_point)
         q2 = q.reshape([15])
         self.assertEqual(q.numel(), q2.numel())
         self.assertEqual(q2.size(), [15])
         # testing -1
         self.assertEqual(q, q2.reshape([3, -1]))
 
-        a = torch._empty_affine_quantized([1, 2, 3, 4], scale=scale, zero_point=zero_point, dtype=dtype)
+        a_int = torch.randint(0, 100, [1, 2, 3, 4], dtype=dtype)
+        a = torch._per_tensor_affine_qtensor(a_int, scale=scale, zero_point=zero_point)
         b = a.transpose(1, 2)  # swaps 2nd and 3rd dimension
-        c = a.reshape(1, 3, 2, 4)  # does not change tensor layout
+        c = a.reshape(1, 3, 2, 4)  # changes tensor layout in memory
```

Comment:
Actually I think this is the same as a view right? @gchanan

Reply:
View take sizes, reshape takes dimension ids

Reply:
I mean the effect should be the same in this case right?

Reply:
doc for reshape says: `Returns a tensor with the same data and number of elements as input, but with the specified shape. When possible, the returned tensor will be a view of input. Otherwise, it will be a copy.`